### PR TITLE
app-shells/autojump: fix Python import error

### PR DESCRIPTION
**Pull request by lukipuki**
Fixes https://bugs.gentoo.org/show_bug.cgi?id=559448

### CODE REVIEW

**Comment by monsieurp** [comment]
Merged :+1:

File: app-shells/autojump/autojump-22.2.4-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        true
+}
+
+src_install() {
+        dobin bin/autojump
+
+        insinto /etc/profile.d
+        doins bin/${PN}.sh
+
+        insinto /usr/local/share/autojump/
```
Comment (lkraav) [inline app-shells/autojump/autojump-22.2.4-r1.ebuild]: Guys, this can't be right. `/usr/local/share`??

Comment (hasufell) [inline app-shells/autojump/autojump-22.2.4-r1.ebuild]: yes, it is not allowed

**Comment by lukipuki** [comment]
Fixed in https://github.com/gentoo/gentoo/pull/87

File: app-shells/autojump/autojump-22.2.4-r1.ebuild (hunk 1 above)
Comment (lukipuki) [inline app-shells/autojump/autojump-22.2.4-r1.ebuild]: Fixed in #87

File: app-shells/autojump/autojump-22.2.4-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        fi
+
+        if use zsh-completion ; then
+                doins bin/${PN}.zsh
+                insinto /usr/share/zsh/site-functions
+                doins bin/_j
+        fi
+
+        if use python ; then
+                python_foreach_impl python_domodule tools/autojump_ipython.py bin/autojump_argparse.py bin/autojump_data.py bin/autojump_utils.py
```
Comment (lkraav) [inline app-shells/autojump/autojump-22.2.4-r1.ebuild]: `python` use flag is also still a mess, but I guess that discussion is on bgo. Short summary: the package still won't work at all with `-python`. Needs to be an implicit dependency. That's `IUSE`, correct?

Comment (lukipuki) [inline app-shells/autojump/autojump-22.2.4-r1.ebuild]: Yes, python is a dependency (https://github.com/wting/autojump), so the python useflag is ambiguous. In #87 , I've made the installtion of the `bin/*.py` files non-optional.

**Pull request closed without merging**